python_apps/api_clients/api_clients/utils.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, name, url, logger=None, api_key=None):
             self.logger = logger
         self.auth = KeyAuth(api_key)
 
-    def __call__(self, _post_data=None, params=None, **kwargs):
+    def __call__(self, _post_data=None, _put_data=None, params=None, **kwargs):
```

Comment:
Let's protect us from future breaking changes and enforce using a keyword based argument passing to this function.

```patch
-    def __call__(self, _post_data=None, _put_data=None, params=None, **kwargs):
+    def __call__(self, *, _post_data=None, _put_data=None, params=None, **kwargs):
```

Reply:
Need to check in the version1.py and version2.py files. Those and their tests should be the only things calling it